### DISCUSSION TITLE
LDAP: ENABLE_LOGIN_FORM=False also disables LDAP form

### DISCUSSION THREAD

**Discussion by strobel1x** [open-webui/open-webui#7172] [General]
When I set `LDAP_ENABLE=True` and `ENABLE_LOGIN_FORM=False`, no LDAP login form is displayed, but the link to switch between the two is displayed.
This should not be the case as discribed in the [commit](https://github.com/open-webui/open-webui/pull/6743/commits/5d934d7d1502891f0d6d6e5f155aba43d456f2fa).

**Comment by morgan55555** [comment]
https://github.com/open-webui/open-webui/pull/7162

**Reply by strobel1x** [reply]
Thanks for the fix and the improvement.

With the latest release (0.4.3), when a new user signs in via LDAP, he gets an error message:
`500: [ERROR: 1 validation error for UserModel profile_image_url Input should be a valid string [type=string_type, input_value=None, input_type=NoneType] For further information visit https://errors.pydantic.dev/2.9/v/string_type]`

It looks like the `None` in the `profile_image_url` parameter [here](url) doesn't get properly set to the default value `/user.png` in [`insert_new_auth`](https://github.com/open-webui/open-webui/blob/c13bcfdfc9fcb9b72e516cb72c89f234511c89df/backend/open_webui/apps/webui/models/auths.py#L98) (or somewhere else on the way to the [`UserModel`](https://github.com/open-webui/open-webui/blob/c13bcfdfc9fcb9b72e516cb72c89f234511c89df/backend/open_webui/apps/webui/models/users.py#L40)

Maybe use named parameters?
```
user = Auths.insert_new_auth(
    email=mail,
    password=str(uuid.uuid4()),
    name=cn,
    role=role,
)
```
